Adyen/UI/Form/Items/Basic Items/FormLabelItem.swift

```diff
@@ -21,24 +22,35 @@ public class FormLabelItem: FormItem {
         self.text = text
     }
 
+    public init(
+        text: String,
+        identifier: String? = nil,
+        labelStyle: LabelStyle = LabelStyle()
+    ) {
+        self.identifier = identifier
+        self.labelStyle = labelStyle
+        self.text = text
+    }
+
     public var identifier: String?
 
     /// The style of the label.
-    public var style: TextStyle
+    public var style: TextStyle = .init(font: .preferredFont(forTextStyle: .body), color: .red)
 
     /// The text of the label.
     public var text: String
 
+    /// The labelStyle from the adyen theme
+    public var labelStyle: LabelStyle = .init()
```

Comment:
`public var style: LabelStyle` reads a bit easier and still provides enough context

Reply:
we already have `style` property in this class to support old styling. We can change the name later once we get rid of the old styling. WDYT?

Reply:
As discussed I will make this change in upcoming PR.